storybook/src/components/FieldSet/FieldSet.stories.tsx

```diff
@@ -38,7 +40,7 @@ type Story = StoryObj<typeof meta>
 
 export const Default: Story = {
   render: (args) => (
-    <FieldSet invalid={args.invalid} legend={args.legend}>
+    <FieldSet invalid={args.invalid} legend={args.legend} hint={args.hint} optional={args.optional}>
```

Comment:
Let’s destructure `args` in cases like this to make the implementation easier to read.